modules/conversion-center/classes/render/render-base.php

```diff
@@ -166,78 +198,217 @@ protected function render_bio(): void {
+					// Escaped above
+					Utils::print_unescaped_internal_string( $bio_about_heading_output );
+				} ?>
 				<?php if ( $has_bio_description ) {
 					$this->widget->add_render_attribute( 'description', 'class', 'e-link-in-bio__description' );
-					$bio_description_output = sprintf( '<p %1$s>%2$s</p>', $this->widget->get_render_attribute_string( 'description' ), $bio_description_value );
+					$bio_description_output = sprintf( '<p %1$s>%2$s</p>', $this->widget->get_render_attribute_string( 'description' ), esc_html( $bio_description_value ) );
+					// Escaped above
 					Utils::print_unescaped_internal_string( $bio_description_output );
 				} ?>
 			</div>
 			<?php
 		}
 	}
 
-	protected function render_identity_image(): void {
-		$identity_image_props_shape = $this->settings['identity_image_shape'] ?? 'circle';
-		$identity_image_props_style = $this->settings['identity_image_style'] ?? 'profile';
-		$identity_image_props_show_border = $this->settings['identity_image_show_border'] ?? false;
-		$identity_image_props_show_bottom_border = $this->settings['identity_image_show_bottom_border'] ?? false;
-		$identity_image_value = $this->settings['identity_image'] ?? [];
+	protected function render_footer_bio(): void {
+
+		if ( 'bottom' !== $this->widget->get_description_position() ) {
+			return;
+		}
+
+		$bio_about_heading_props_tag = $this->settings['bio_about_tag'] ?? 'h3';
+		$bio_about_heading_value = $this->settings['bio_about'] ?? '';
+
+		$bio_description_value = $this->settings['bio_description'] ?? '';
+
+		$has_bio_description = ! empty( $bio_description_value );
+		$has_bio_about_heading = ! empty( $bio_about_heading_value );
+
+		if ( $has_bio_about_heading || $has_bio_description ) {
+			?>
+			<div class="e-link-in-bio__bio e-link-in-bio__bio--footer">
+				<?php if ( $has_bio_about_heading ) {
+					$this->widget->add_render_attribute( 'about-heading', 'class', 'e-link-in-bio__about-heading' );
+					$bio_about_heading_output = sprintf( '<%1$s %2$s>%3$s</%1$s>', Utils::validate_html_tag( $bio_about_heading_props_tag ), $this->widget->get_render_attribute_string( 'about-heading' ), esc_html( $bio_about_heading_value ) );
+					// Escaped above
+					Utils::print_unescaped_internal_string( $bio_about_heading_output );
+				} ?>
+				<?php if ( $has_bio_description ) {
+					$this->widget->add_render_attribute( 'description', 'class', 'e-link-in-bio__description' );
+					$bio_description_output = sprintf( '<p %1$s>%2$s</p>', $this->widget->get_render_attribute_string( 'description' ), esc_html( $bio_description_value ) );
+					// Escaped above
+					Utils::print_unescaped_internal_string( $bio_description_output );
+				} ?>
+			</div>
+			<?php
+		}
+	}
+
+	protected function get_shape_divider( $side = 'bottom' ) {
```

Comment:
Do we need to add code for the shape divider? Isn't this an existing feature? Couldn't this be handled e.g. in a trait? https://github.com/elementor/elementor/blob/9736f1ebbbadfc47716be005392969e4423b1eba/includes/elements/container.php#L226

Reply:
@hein-obox agreed however this is already duplicated and may be better managed as a separate task
see 
 https://github.com/elementor/elementor/blob/9736f1ebbbadfc47716be005392969e4423b1eba/includes/elements/section.php#L1557

Reply:
@hein-obox moved to a trait